source/detail/environment.hpp

```diff
@@ -79,9 +80,10 @@
 /// <seealso cref="XLNT_C_23">
 #define XLNT_HAS_C_VERSION(version) (1/version == 1/version && \
     ((__STDC_VERSION__ >= version) || \
-     (version == XLNT_C_11 && XLNT_HAS_CPP_VERSION(XLNT_CPP_17)) || \
-     (version == XLNT_C_17 && XLNT_HAS_CPP_VERSION(XLNT_CPP_20)) /*|| \
-     (version == CLNT_C_23 && XLNT_HAS_CPP_VERSION(XLNT_CPP_26))*/ \
+     (version >= XLNT_C_99 && XLNT_HAS_CPP_VERSION(XLNT_CPP_11)) || \
```

Comment:
I looked again at this check, and I think the check is wrong.
If i want to check if C_11 is supported, I would call this macro with version = C_11.  So, version >= XLNT_C_99 and if I have a C++11 compiler `XLNT_HAS_CPP_VERSION(XLNT_CPP_11)` is true too. So, this macro will say that C_11 is supported, which is not the case for a (default) C++11 compiler.

Reply:
You are right. This is what I get from committing late in the evening, I guess :sweat_smile: Including typos in commit messages. Anyway, I fixed this in my newest commit.